repository/repository.py

```diff
@@ -0,0 +1,36 @@
+from database.models.AutoModel import AutoModel
+from database.models.MotoModel import MotoModel
+from database.models.TruckModel import TruckModel
+from database.enums.TransportType import TransportType
+
+
+def get_transport_query(db, args, type):
```

Comment:
we can name it same with service functions, get_transport enough, it can be query, it can be request for another api, it doesnt matter